### PR TITLE
Add deploy-7 docs

### PR DESCRIPTION
**Pull request by gomathyk**

### CODE REVIEW

File: en/docs/deploy/monitor/http-access-logging.md [hunk 1]
```diff
@@ -2,21 +2,18 @@
 
 HTTP access logs help you monitor your application's usage with
 information such as the persons who access it, how many hits
-it received, what the errors are, etc. This information is useful for
+it receives, what the errors are, etc. This information is useful for
 troubleshooting errors. WSO2 Identity Server can enable access logs for the
 HTTP servlet transport. This servlet transport works on `9443`/`9763` ports,
-and it recieves admin/operation requests. Therefore, access logs for the
+and it receives admin/operation requests. Therefore, access logs for the
 servert transpot is useful for analysing operational/admin-level access
```
Comment (dewniMW) [inline en/docs/deploy/monitor/http-access-logging.md:9]:
```suggestion
servlet transport is useful for analyzing operational/admin-level access
```

File: en/docs/deploy/monitor/http-access-logging.md [hunk 2]
```diff
@@ -37,27 +34,27 @@ In the Identity Server 5.9.0 only the access log pattern is configurable.
     <tbody>
     <tr class="odd">
     <td>directory</td>
-    <td>The path to the directory that will store the access log file. By default, this is location is set to <code>               ${carbon.home}/repository/logs              </code> in all WSO2 products.</td>
+    <td>This is the path to the directory that will store the access log file. By default, this location is set to <code>               ${carbon.home}/repository/logs              </code> for WSO2 Identity Server.</td>
```
Comment (dewniMW) [inline en/docs/deploy/monitor/http-access-logging.md:37]:
```suggestion
    <td>This is the path to the directory that will store the access log file. By default, this location is set to <code>               <IS_HOME>/repository/logs              </code> for WSO2 Identity Server.</td>
```

File: en/docs/deploy/monitor/monitor-server-health.md [hunk 3]
```diff
@@ -1,37 +1,30 @@
...
         80%) of the maximum allowed connections count. This checker also
-        tests the connection from each data source to see whether the
+        tests the connection from each datasource to see whether the
         connection is successful.
     2.  **Server startup health checker** - This checker uses the
-        ServerAdmin service to check if the server status is RUNNING.
-    3.  **Super tenant user store health checker** - This checker iterates
-        through configured user stores of the super tenant domain and
+        `ServerAdmin` service to check if the server status is RUNNING.
+    3.  **Super tenant userstore health checker** - This checker iterates
```
Comment (dewniMW) [inline en/docs/deploy/monitor/monitor-server-health.md:17]:
```suggestion
    3.  **Super tenant userstore health checker**: This checker iterates
```

use `:` in all applicable places

File: en/docs/deploy/monitor/system-statistics.md [hunk 4]
```diff
@@ -21,54 +20,54 @@ response time.
...
 
--   **Host** - Shows the hostname of the server.
--   **Server Start Time** - Shows the time when the server started.
+-   **Host** - Shows the hostname of the server
+-   **Server Start Time** - Shows the time when the server started
 -   **System Up Time** - Shows the amount of time that the server has
-    been working.
--   **Memory Allocated** - Shows the memory capacity of the server.
--   **Memory Usage** - Shows the memory capacity used by the server.
+    been working.\
```
Comment (dewniMW) [inline en/docs/deploy/monitor/system-statistics.md:50]:
```suggestion
    been working.
```

File: en/docs/deploy/monitor/system-statistics.md [hunk 5]
```diff
@@ -78,14 +77,14 @@ Use the **Statistics Configuration** panel to configure the statistics
 view.
 
 1.  Enter values into the appropriate fields:
-    -   **Statistics Refresh Interval (ms)** - Allows you to specify the
-        statistics refresh rate.
-    -   **Response Time Graph** - Allows you to specify the X and Y
-        parameters of the Response Time graph.
+    -   **Statistics Refresh Interval (ms)** - Allows to specify the
```
Comment (dewniMW) [inline en/docs/deploy/monitor/system-statistics.md:80]:
```suggestion
    -   **Statistics Refresh Interval (ms)**: Allows to specify the
```

File: en/docs/deploy/monitor/monitor-server-health.md (hunk 3 above)
Comment (gomathyk) [reply en/docs/deploy/monitor/monitor-server-health.md:17]: We need to look for the best practice here.

File: en/docs/deploy/monitor/system-statistics.md (hunk 5 above)
Comment (gomathyk) [reply en/docs/deploy/monitor/system-statistics.md:80]: Refer https://github.com/wso2/docs-is/pull/2645#discussion_r645517587

**Review by dewniMW — APPROVED** [review]

**Pull request merged**